### DISCUSSION TITLE
URL to my pages is username.github.io/username.github.io

### DISCUSSION THREAD

**Discussion by robertbyers1111** [community/community#34908] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm trying to set up my personal github pages site per the getting started instructions at pages.github.com. The final step indicates the URL for my pages should be https://username.github.io ... however that gives me a 404 error. After looking at my repository's settings (settings > code and automation > Pages) I notice the message..

Your site is live at https://robertbyers1111.github.io/robertbyers1111.gitub.io/

... but that isn't what I was expecting. According to the instructions it should have been https://robertbyers1111.github.io/ (i.e., without the redundant "robertbyers1111.github.io")

I cannot see where I deviated from the instructions. Any suggestions?

**Comment by antonblr** [comment, marked as answer]
@robertbyers1111  - You have typo in the repo name: `robertbyers1111.gitub.io` -> `robertbyers1111.github.io` (missing "**h**")

**Reply by robertbyers1111** [reply]
I need to have my eyes checked!!! Thanks!

**Reply by antonblr** [reply]
Sure (*EDIT:* meaning you're welcome, not that you need your eyes checked ))! Please update it to remove "Unanswered" status.

**Reply by judeking20001210-ux** [reply]
hmm

**Comment by subupatayach** [comment]
gmail, and all mail concert my mail id account acctiovets online support
and all world all contrary all bank accounts actiovtes on online 0balance
acctiovets and online international transaction money created my mail 🆔
accounts actiovtes and segunail tower setelight everything online include
support my mail 🆔 accounts on online.

**Comment by baraa639629402-ui** [comment]
B.R.A School | دورات تعلم البرمجة من الصفر

**Comment by baraa639629402-ui** [comment]
B.R.A School | دورات تعلم البرمجة من الصفر

**Comment by digital12759** [comment]
سلام اشتباه تایپ دارید در قسمت جستجو به این معنی که کلمه GitHub را درست نوشتید به غلط Gitubنوشتید
